### ISSUE TITLE
The module 'react-dom' was not found

### ISSUE THREAD

**Issue by jakeherp** [zeit/next.js#8318]
# Bug report

`next dev` and `next build` are not working for me since the upgrade to React 16.9.0 with Next 9.0.3.
The error I get is:

```
The module 'react-dom' was not found. Next.js requires that you include it in 'dependencies' of your 'package.json'. To add it, run 'npm install --save react-dom'
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
Error: Cannot find module 'react-dom/server'
Require stack:
- /Users/jacob/.config/yarn/global/node_modules/next-server/dist/server/render.js
- /Users/jacob/.config/yarn/global/node_modules/next-server/dist/server/next-server.js
- /Users/jacob/.config/yarn/global/node_modules/next/dist/server/next-dev-server.js
- /Users/jacob/.config/yarn/global/node_modules/next/dist/server/next.js
- /Users/jacob/.config/yarn/global/node_modules/next/dist/server/lib/start-server.js
- /Users/jacob/.config/yarn/global/node_modules/next/dist/cli/next-dev.js
- /Users/jacob/.config/yarn/global/node_modules/next/dist/bin/next
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/jacob/.config/yarn/global/node_modules/next-server/dist/server/render.js:14:18)
    at Module._compile (internal/modules/cjs/loader.js:777:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:788:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:683:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/jacob/.config/yarn/global/node_modules/next-server/dist/server/render.js',
    '/Users/jacob/.config/yarn/global/node_modules/next-server/dist/server/next-server.js',
    '/Users/jacob/.config/yarn/global/node_modules/next/dist/server/next-dev-server.js',
    '/Users/jacob/.config/yarn/global/node_modules/next/dist/server/next.js',
    '/Users/jacob/.config/yarn/global/node_modules/next/dist/server/lib/start-server.js',
    '/Users/jacob/.config/yarn/global/node_modules/next/dist/cli/next-dev.js',
    '/Users/jacob/.config/yarn/global/node_modules/next/dist/bin/next'
  ]
}
```

I have tried a `yarn global upgrade` to make sure everything is up-to-date on my machine, but keep getting this error. Running `npx next dev` works fine.

## To Reproduce

Fresh installation of Next.js 9.0.3 with React and React DOM at version 16.9 shows the same behaviour.

## System information

- OS: macOS 10.14.6
- Command-line tool: ZSH
- Version of Next.js: 9.0.3
- Version of React / React DOM: 16.9.0

**Comment by timneutkens** [comment]
I can't reproduce this.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
